tools/PI/DevHome.PI/BarWindow.cs

```diff
@@ -118,6 +122,21 @@ private void ViewModel_PropertyChanged(object? sender, PropertyChangedEventArgs
         }
     }
 
+    public void ShowDialogToEnableAppExecutionAlias()
+    {
+        _ = _horizontalWindow.ShowMessageDialogAsync(dialog =>
```

Comment:
Change the access modifier of `ShowMessageDialogAsync` tells me something is wrong.  Can `ShowMessageDialog` go into a common location?

Reply:
Would you recommend any other changes in the WindowExExtensions.cs file?